Cpp/source/DrawingEngine/DrawingEngine.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <iostream>
+// #include <cstdint>
+#include <stdint.h>
+#define BUFFER_SIZE 20
```

Comment:
it would be nice if we could pass size in the constructor

Reply:
done